MARDEK Engine/Assets/Scripts/MARDEK/Inventory/Item/JSONReader.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using FullSerializer;
+using MARDEK.Core;
+using MARDEK.Stats;
+using MARDEK.Skill;
+
+namespace MARDEK.Inventory
+{
+    public class JSONReader : MonoBehaviour
```

Comment:
This class should be named something like PlotItemImporter

Reply:
Renamed to ItemImporter, since it can import any type of item, not just plot items.